app/src/main/java/de/westnordost/streetcomplete/quests/diet_type/AddKosher.kt

```diff
@@ -0,0 +1,34 @@
+package de.westnordost.streetcomplete.quests.diet_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+
+class AddKosher : OsmFilterQuestType<String>() {
+
+    override val elementFilter = """
+        nodes, ways with
+        (
+          amenity ~ restaurant|cafe|fast_food
+          or amenity = ice_cream
+          or shop = butcher
```

Comment:
shop = supermarket too and bakery:
https://www.kosher.com/learn/about-kosher-laws/kosher-bread

Reply:
Did the OSM Israel people suggest bakery wasn't worthwhile then @matkoniecz ?